### PR TITLE
[runtime] Remove a mono_error_raise_exception () call from mono_remoting_wrapper (), its not needed since mono_remoting_wrapper () is called through a wrapper.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
The failures are caused by this PR.

**Comment by vargaz** [comment]
build

File: mono/metadata/threads.c [hunk 1]
```diff
@@ -4598,7 +4598,7 @@ mono_thread_interruption_checkpoint_request (gboolean bypass_abort_protection)
 		return NULL;
 	if (!mono_thread_get_interruption_requested (thread))
 		return NULL;
-	if (!bypass_abort_protection && is_running_protected_wrapper ())
+	if (!bypass_abort_protection && !mono_thread_current ()->pending_exception && is_running_protected_wrapper ())
```
Comment (BrzVlad) [inline mono/metadata/threads.c]: @vargaz Wasn't the previously set pending exception getting thrown from the managed to native wrapper ?

Comment (vargaz) [reply mono/metadata/threads.c]: Yes, but is_running_protected_wrapper () does a full stack walk, so it won't throw if there is a remoting wrapper above, hence this change.

**Pull request merged**